source/_docs/scripts/conditions.markdown

```diff
@@ -241,7 +241,7 @@ A better weekday condition could be by using the [Workday Binary Sensor](/integr
 
 ### Zone condition
 
-Zone conditions test if an entity is in a certain zone. For zone automation to work, you need to have setup a device tracker platform that supports reporting GPS coordinates. Currently this is limited to the [OwnTracks platform](/integrations/device_tracker.owntracks/) and the [iCloud platform](/integrations/device_tracker.icloud/).
+Zone conditions test if an entity is in a certain zone. For zone automation to work, you need to have setup a device tracker platform that supports reporting GPS coordinates. Currently this is limited to the [OwnTracks platform](/integrations/device_tracker.owntracks/) and the [iCloud platform](/integrations/icloud/).
```

Comment:
For stuff like that we have already [redirects](https://github.com/home-assistant/home-assistant.io/blob/next/source/_redirects#L339).

Reply:
Does not mean that this adjustment is not correct, once it has changed everywhere, the redirect is no longer necessary 😉

Reply:
But that means that the link should work.
If i click on [iCloud platform](https://www.home-assistant.io/integrations/device_tracker.icloud/) in https://www.home-assistant.io/docs/scripts/conditions/#zone-condition I get redirected to Page Not Found.

Reply:
Same with https://www.home-assistant.io/docs/mqtt/broker/#cloudmqtt
 [OwnTracks](https://www.home-assistant.io/integrations/device_tracker.owntracks)

Reply:
While not broken, It is OK (and personally to me preferred) to change those. If we can reduce redirects, that is always a win. Both for client<->server round trips, but also SEO wise.